Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/presencetracing/organizer/details/QrCodeDetailViewModel.kt

```diff
@@ -86,9 +97,15 @@ class QrCodeDetailViewModel @AssistedInject constructor(
         }
     }
 
-    fun openFullScreen() {
+    fun openFullScreen() = launch {
         traceLocation?.let {
-            routeToScreen.postValue(QrCodeDetailNavigationEvents.NavigateToFullScreenQrCode(it.locationUrl))
+            val correctionLevel = appConfigProvider.getAppConfig().presenceTracing.qrCodeErrorCorrectionLevel
```

Comment:
Do we need `getAppConfig`? Would the more friendly `latestAppConfig` also suffice?

Reply:
i used what was there before, Maybe @axelherbstreith  can answer that?

Reply:
There is no `latestAppConfig`

Reply:
Same as @mtwalli, I also reused the implementation of the QR code which was already there. I can't find `latestAppConfig`, do you maybe mean `currentAppConfig`? @d4rken